source/common/common/utility.cc

```diff
@@ -192,9 +192,9 @@ void DateFormatter::parse(const std::string& format_string) {
   }
 }
 
-std::string
-DateFormatter::fromTimeAndPrepareSpecifierOffsets(time_t time, SpecifierOffsets& specifier_offsets,
-                                                  const std::string& seconds_str) const {
+std::string DateFormatter::fromTimeAndPrepareSpecifierOffsets(time_t time,
+                                                              SpecifierOffsets& specifier_offsets,
+                                                              absl::string_view seconds_str) const {
```

Comment:
To preserve original constness, please consider.

```suggestion
                                                              const absl::string_view seconds_str) const {
```

Reply:
sure